### PR TITLE
use real live DOM elements and get rid of selectize

### PR DESCRIPTION
**Pull request by brenna**

### CODE REVIEW

**Comment by brenna** [comment]
@samselikoff @ryanto I think you've both seen most of this code already but for the sake of getting into the habit, can i get a :+1: on this before merging?

File: addon/components/ted-select.js [hunk 1]
```diff
@@ -1,118 +1,59 @@
...
-        this.$('select')[0].selectize.close();
+    onChange(target){
+      let value = $(target).val(),
+          selection;
+
+      //if multiple, .val() returns an array. if not, it's a single value
+      if (this.get('multiple')){
+        selection = this.get('content').filter(option => {
+          let optionVal = Ember.get(option, this.get('optionValueKey')).toString();
+          return Ember.A(value).contains(optionVal);
```
Comment (ryanto) [inline addon/components/ted-select.js]: Might want to do the Ember.A call outside the loop.

**Comment by ryanto** [comment]
Looks good :+1:

**Comment by samselikoff** [comment]
nice, lgtm. great work on this!

File: addon/components/ted-select.js (hunk 1 above)
Comment (brenna) [inline addon/components/ted-select.js]: good call. fixed.

**Pull request merged**